crates/suiop-cli/src/cli/incidents/pd.rs

```diff
@@ -251,37 +260,47 @@ pub async fn print_recent_incidents(
     Ok(())
 }
 
-pub async fn review_recent_incidents(incidents: Vec<Incident>) -> Result<()> {
-    let incidents_grouped = incidents
+/// Filter incidents based on whether they have > min_priority priority or any slack
+/// channel associated.
+fn filter_incidents_for_review(incidents: Vec<Incident>, min_priority: &str) -> Vec<Incident> {
+    incidents
         .into_iter()
         // filter on priority > P3 and any slack channel association
         .filter(|i| {
             i.priority
                 .clone()
-                .filter(|p| !p.name.is_empty() && p.name != "P3")
+                .filter(|p| !p.name.is_empty() && p.name != min_priority)
```

Comment:
nit: so this one is `!=`, which works for P3 the lowest priority. But if we pass `P2` here thing will be broken right? Maybe let's cap the string and use `<`? that should work

Reply:
Ah good catch, will fix it!